NangPaGo-api/NangPaGo-admin/src/main/java/com/mars/admin/domain/dashboard/controller/DashboardController.java

```diff
@@ -19,4 +19,10 @@ public class DashboardController {
     public ResponseDto<Map<String, Long>> dashboard() {
         return ResponseDto.of(chartService.getTotals(), "");
     }
-}
+
+
+    @GetMapping("/month/post")
```

Comment:
월단위 통계라서 `/month` 를 사용하신걸까요

'개월'이라는 개념 자체를 '자원'으로 보기는 어려운 것 같아요. 다른 이름이 좋을 것 같아요.
```suggestion
    @GetMapping("/stats/post")
```

할 수 있을지는 모르겠지만, 추후 월단위가 아닌 일단위, 연단위 통계도 볼 수 있게 확장한다면 `/stats/post?unit=monthly` 와 비슷한 방식으로 구현할 수 있게 될 것 같네요

Reply:
수정했습니다.